### PR TITLE
Update FL-route design documents

### PR DESCRIPTION
**Pull request by surabujin**

### CODE REVIEW

File: docs/design/fl-parallelism/README.md [hunk 1]
```diff
@@ -7,83 +7,101 @@
...
-One significant change will be made for message processing - each message
-will have a response9at least one). In other word FL must send response
-for each message(command) it receive. It can be a "normal" response or
-error response. If FL do not produce response, FL-router will produce
-error response by himself.
+Each speaker-region is represented by speaker instance and the set of
+"incoming" and the set of "outgoing" kafka-topics. Region's name is used as the
+suffix for all these kafka-topic names.
+
+FL-router can be treated as the set of parallel streams responsible for poxing
```
Comment (niksv) [inline docs/design/fl-parallelism/README.md:14]: pRoxing?

Comment (rtretyak) [reply docs/design/fl-parallelism/README.md:14]: pRoxYing then

File: docs/design/fl-parallelism/fl-alive-tracking.puml [hunk 2]
```diff
@@ -11,10 +11,6 @@ end box
 participant "FL-router" as fl_router
```
Comment (timofei-durakov) [inline docs/design/fl-parallelism/fl-alive-tracking.puml:11]: nit: as for me I'd change flr and fls on a diagram to read it from left to right

Comment (surabujin) [reply docs/design/fl-parallelism/fl-alive-tracking.puml]: done

**Pull request merged**